qiskit_nature/transformations/bosonic_transformation.py

```diff
@@ -22,9 +22,9 @@
 import numpy as np
 
```

Comment:
You can remove the entire `qiskit_nature.transformations` module as all of this is being refactored :+1: